docs/src/reference/changes.md

```diff
@@ -10,11 +10,25 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## v0.4.0
+
+## Added
 - Updated documentation, now using VitePress.
 - Add new driver framework to read files using native drivers.
 - Added native driver package extensions on GeoJSON, ShapeFile, GeoParquet, GeoArrow and FlatGeobuf.
 - Geometry columns are now wrapped in a GeometryVector, allowing for future improvements.
-- Now exports GeoInterface methods instead of ArchGDAL methods for geometry operation.
+
+### Changed
+- Now exports GeometryOps and GeoInterface methods instead of ArchGDAL methods for geometry operation.
+- The `geom_columns` keyword for `write` is now `geometrycolumn`, and also accepts a single Symbol.
+
+### Removed
+- Removed automatic broadcast implementations of operations like buffer.
```

Comment:
should we still do the manual broadcast implementation on these functions, so that `buffer.(df, num)` just runs `apply`?  should that be a dispatch on dataframes, or any table?

Reply:
Yeah GO kind of does auto broadcasting

Reply:
with apply

Reply:
We should give you a list of table compatible methods.

The rest will not work.